bitnami/elasticsearch/README.md

```diff
@@ -436,6 +436,12 @@ Find more information about how to deal with common errors related to Bitnami’
 
 ## Upgrading
 
+### To 15.0.0
+
+From this version onwards, Elasticsearch container components are now licensed under the Elastic License that is not currently accepted as an Open Source license by the Open Source Initiative (OSI).
+
+Also, from now on, the Helm Chart will include the X-Pack plugin installed by default.
```

Comment:
Should I expect issue upgrading from 14 to 15? Please document that.

Reply:
No issues. I added a comment in the note about that.